examples/amp-story/interactives.html

```diff
@@ -16,6 +16,7 @@
     </style>
     <script async src="https://cdn.ampproject.org/v0.js"></script>
     <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async src="https://cdn.ampproject.org/v0/amp-story-interactive-1.0.js"></script>
```

Comment:
Does this mean a single extension provides multiple CustomElement definitions? Is this a pattern we want across the codebase?

Reply:
Good question. `amp-story` currently has 10+ CustomElement definitions, we're trying to move some of these to their own extensions to optimize bundle size.
All interactive components (quiz, binary poll, poll) support the exact same configuration, options, and styling mechanisms. We'll document them together. They're so similar that we debated having different tagNames vs having an extra attribute, which is why we'd like to keep them together.
Let us know if you think that makes sense.

Reply:
What's the size of the interactive extension? Coupling them together has long term consequences for the binary size of the 'interactive' bundle. Do you expect to grow the number and capabilities in a way that will become detrimental to couple them together?

I'd lean toward keeping them separate, but allowing for optimizers (like AMP Optimizer or AMP Caches) to bundle them together based on future heuristics.

Reply:
To answer the question, the extension is currently 6.7kb removed from amp-story (about 10% of it), but we might add some more interactive components (very similar but different UI), or new styles that add more css to it.

We don't have any metrics yet, because it will take some time before people start using these components, but we plan to check after this PR on how to reduce the size even more